### PR TITLE
feat: adding like to comment and remove reaction methods added

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Adding `addLikeToComment` and `removeReaction` methods to kv-activity-feed.

### CODE REVIEW

File: @kiva/kv-activity-feed/src/index.ts [hunk 1]
```diff
@@ -108,4 +108,42 @@ export default class ActivityFeedService {
...
 		return success;
 	}
+
+	/**
+	 * Adds a like comment for the current user for the provided activity ID
+	 *
+	 * @param commentId The comment ID to add a like
+	 * @returns Whether the like was added successfully
+	 */
+	async addLikeToComment(commentId: string): Promise<boolean> {
```
Comment (dyersituations) [inline @kiva/kv-activity-feed/src/index.ts:118]: Please add unit tests for these new methods, following the pattern of the other methods in this file.

Comment (roger-in-kiva) [reply @kiva/kv-activity-feed/src/index.ts:118]: For this we need to update the `addComment` to return a valid commentId and add a child reaction then.
Said that, we also need to update the current flow we already have in CPS. Makes sense?

Comment (dyersituations) [reply @kiva/kv-activity-feed/src/index.ts:118]: The unit test just needs to verify that we are using the correct 3rd party method, a real quick addition, like my most recent PR.

Comment (roger-in-kiva) [reply @kiva/kv-activity-feed/src/index.ts:118]: Gotcha, thanks for checking.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**